pytorch_lightning/loggers/tensorboard.py

```diff
@@ -147,15 +150,11 @@ def save(self) -> None:
             dir_path = self.save_dir
 
         # prepare the file path
-        meta_tags_path = os.path.join(dir_path, self.NAME_CSV_TAGS)
+        hparams_file = os.path.join(dir_path, self.NAME_HPARAMS_FILE)
 
         # save the metatags file
-        with open(meta_tags_path, 'w', newline='') as csvfile:
-            fieldnames = ['key', 'value']
-            writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
-            writer.writerow({'key': 'key', 'value': 'value'})
-            for k, v in self.tags.items():
-                writer.writerow({'key': k, 'value': v})
+        with open(hparams_file, 'w', newline='') as f:
+            yaml.dump(self.hparams, f)
```

Comment:
I would encapsulate this to a function next to the load to be clear it is symmetric and add doctest with saving and load are equal...